test/common/ssl/connection_impl_test.cc

```diff
@@ -1,17 +1,17 @@
+#include "common/ssl/connection_impl.h"
```

Comment:
nit: Can you put newline between "related file" and the other headers. Same below.